### PR TITLE
Add ID on appsettingsection

### PR DESCRIPTION
**Pull request by GretaD**

### CODE REVIEW

File: src/components/AccountSettings.vue [hunk 1]
```diff
@@ -155,6 +164,19 @@ export default {
 		},
 	},
 	methods: {
+		onAccountUpdated() {
```
Comment (GretaD) [inline src/components/AccountSettings.vue:167]: this is a regresssion from another old pr, and fixed it here for time reasons

File: src/components/AccountSettings.vue [hunk 2]
```diff
@@ -24,9 +24,11 @@
 
 <template>
 	<AppSettingsDialog
+		id="app-settings-dialong"
```
Comment (miaulalala) [inline src/components/AccountSettings.vue:27]:
```suggestion
		id="app-settings-dialogue"
```

**Review by miaulalala — CHANGES_REQUESTED** [review]

File: src/components/AccountSettings.vue (hunk 2 above)
Comment (miaulalala) [reply src/components/AccountSettings.vue:27]: or dialog, whatever is better for you :)

**Review by miaulalala — APPROVED** [review]
:shipit:

File: src/components/AccountSettings.vue (hunk 1 above)
Comment (kesselb) [reply src/components/AccountSettings.vue]: Thanks!

**Pull request merged**